src/NServiceBus.Core/Recoverability/Faults/FaultNotificationConfigExtensions.cs

```diff
@@ -0,0 +1,51 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Collections.Generic;
+    using NServiceBus.Faults;
+    using NServiceBus.Settings;
+
+    /// <summary>
+    /// Provides config options for the faults feature.
+    /// </summary>
+    public static class FaultNotificationConfigExtensions
+    {
+
+        /// <summary>
+        /// Allows for customization of the faults.
+        /// </summary>
+        /// <param name="busConfiguration">The <see cref="BusConfiguration"/> instance to apply the settings to.</param>
+        public static FaultsSettings Faults(this BusConfiguration busConfiguration)
```

Comment:
It depends on what future plans we have. Do we aim for a root `.Recoverability()` or do we have 2 sub roots `.Faults()` and `.Retries()` ? (not in scope for this pull)